### DISCUSSION TITLE
Does this extension support ripping dimensional lumber?

### DISCUSSION THREAD

**Discussion by PavelGutin** [lairdubois/lairdubois-opencutlist-sketchup-extension#414] [Q&A] [answered]
I intend to use 8ft long 2x12 boards in my build, but I plan to rip them to a different width. Is there a way to specify this?

**Comment by mobilarte** [comment, marked as answer]
Short answer is no: dimensional lumber is meant to be used as is. The distinction between dimensional (only cut to length), sheet goods (length and width cuts, but no thickness changes) and rough lumber (part must fit somehow into a board) allows us to add special functions to each category.

If your selection of widths is limited, I would add those dimensions as standard dimensions and use them as if they could be bought as is. The Parts List will then give you a list of necessary boards. You can then rip them from 2 x 12 or 2 x 8 (if they are narrower). I am not familiar with framing lumber, therefore I don't know if wider boards may be of lesser quality or not.

Another possibility, if you are making a lot of parts inside a 2 x 12, would be to configure the material as Sheet Good.

You could even configure the boards as rough lumber. In all three cases, you will lose some functionality.
